src/main/java/dinkplugin/notifiers/LootNotifier.java

```diff
@@ -36,13 +44,27 @@ protected String getWebhookUrl() {
         return config.lootWebhook();
     }
 
+    public void onHitsplat(HitsplatApplied event) {
+        if (event.getHitsplat().isMine() && event.getActor() != client.getLocalPlayer())
+            lastTarget = new WeakReference<>(event.getActor());
+    }
+
     public void onNpcLootReceived(NpcLootReceived event) {
         if (isEnabled())
             this.handleNotify(event.getItems(), event.getNpc().getName());
     }
 
     public void onPlayerLootReceived(PlayerLootReceived event) {
-        // TODO: player loot should be a configurable toggle
+        if (!config.includePlayerLoot())
+            return;
+
+        boolean pk = client.getLocalPlayer().getInteracting() == event.getPlayer() && event.getPlayer() == lastTarget.get();
+        if (!pk && config.restrictPlayerLootByDamage())
```

Comment:
this may be overengineered now; the intent of the event *is* PK, so we probably don't need a config setting here

also, this extra logic is only necessary *if* there are scenarios with multiway pvp combat that can show dropped items from pk to multiple people in the death tick (where not all of these people were actually attacking on the death tick). i'm not sure if this scenario exists, so we can revert this commit if desired